build.gradle

```diff
@@ -0,0 +1,57 @@
+plugins {
+    id 'java-library'
```

Comment:
java-library, signing, maven-publish and io.codearte.nexus-staging - are not needed as a plugins really, as this is not library but just a 'normal' project

Reply:
True, only `java` and `spotless` plugins left.